x-pack/plugins/security_solution/public/users/pages/users.tsx

```diff
@@ -191,7 +197,9 @@ const UsersComponent = () => {
 
             <EuiSpacer />
 
-            <SecuritySolutionTabNavigation navTabs={navTabsUsers} />
+            <SecuritySolutionTabNavigation
+              navTabs={navTabsUsers(hasMlUserPermissions(capabilities), riskyUsersFeatureEnabled)}
```

Comment:
Before `navTabsUsers` was a static object, now it is a new object every time we call the function, shouldn't we memoize it?